### PR TITLE
Added a note about relationships need to be restarted to show in Grap…

### PR DESCRIPTION
**Pull request by itdependsnetworks**
Please indicate the relevant feature request or bug report below.
# Closes: DNE
# What's Changed

Minor doc update

# TODO

- [x] Explanation of Change(s)
- [x] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [x] Documentation Updates (when adding/changing features)

### CODE REVIEW

**Review by glennmatthews — COMMENTED** [review]
Would you mind adding a similar note regarding individual custom fields (the `cf_<name>` case, not the `custom_field_data` case)?

**Comment by itdependsnetworks** [comment]
Updated, plus added docs for computed field as well.

File: nautobot/docs/additional-features/graphql.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ Result
 }
 ```
 
-Additionally, by default, all custom fields in GraphQL will be prefixed with `cf`. A custom field name `site_type` will appear in GraphQL as `cf_site_type` as an example. The prefix can be changed or remove the prefix by setting the value of `GRAPHQL_CUSTOM_FIELD_PREFIX`.
+Additionally, by default, all custom fields in GraphQL will be prefixed with `cf_`. A custom field name `site_type` will appear in GraphQL as `cf_site_type` as an example. The prefix can be changed or remove the prefix by setting the value of `GRAPHQL_CUSTOM_FIELD_PREFIX`.
```
Comment (glennmatthews) [inline nautobot/docs/additional-features/graphql.md:104]: Wouldn't be a bad idea to add a cross-link here:

```suggestion
Additionally, by default, all custom fields in GraphQL will be prefixed with `cf_`. A custom field name `site_type` will appear in GraphQL as `cf_site_type` as an example. The prefix can be changed by setting the value of [`GRAPHQL_CUSTOM_FIELD_PREFIX`](../configuration/optional-settings.md#graphql_custom_field_prefix).
```

File: nautobot/docs/additional-features/graphql.md [hunk 2]
```diff
@@ -202,6 +208,44 @@ Result
 }
 ```
 
+## Working with Computed Fields
+
+By default, all custom fields in GraphQL will be prefixed with `cpf_`. A computed field name `ip_ptr_record` will appear in GraphQL as `cpf_ip_ptr_record` as an example. The prefix can be changed or remove the prefix by setting the value of `GRAPHQL_COMPUTED_FIELD_PREFIX`.
```
Comment (glennmatthews) [inline nautobot/docs/additional-features/graphql.md:213]:
```suggestion
By default, all custom fields in GraphQL will be prefixed with `cpf_`. A computed field name `ip_ptr_record` will appear in GraphQL as `cpf_ip_ptr_record` as an example. The prefix can be changed by setting the value of [`GRAPHQL_COMPUTED_FIELD_PREFIX`](../configuration/optional-settings.md#graphql_computed_field_prefix).
```

File: changes/2619.added [hunk 3]
```diff
@@ -0,0 +1 @@
+Added notes about relationships, custom fields, and computed fields need to be restarted to show in GraphQL.
```
Comment (glennmatthews) [inline changes/2619.added:1]:
```suggestion
Added notes about restarting the Nautobot server after creating relationships, custom fields, and computed fields to show them in GraphQL.
```

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**